packages/patternfly-4/react-core/src/components/Breadcrumb/BreadcrumbHeading.tsx

```diff
@@ -1,33 +1,28 @@
-import React from 'react';
+import * as React from 'react';
 import styles from '@patternfly/patternfly/components/Breadcrumb/breadcrumb.css';
-import PropTypes from 'prop-types';
 import { css } from '@patternfly/react-styles';
-import { componentShape } from '../../helpers/componentShape';
 
-const propTypes = {
+export interface BreadcrumbHeadingProps extends React.HTMLProps<HTMLLIElement> {
   /** Content rendered inside the breadcrumb title. */
-  children: PropTypes.node,
+  children?: React.ReactNode;
   /** Additional classes added to the breadcrumb item. */
-  className: PropTypes.string,
+  className?: string;
   /** HREF for breadcrumb link. */
-  to: PropTypes.string,
+  to?: string;
   /** Target for breadcrumb link. */
-  target: PropTypes.string,
+  target?: string;
   /** Sets the base component to render. Defaults to <a> */
-  component: componentShape,
-  /** Additional props are spread to the container <li> */
-  '': PropTypes.any
-};
+  component?: React.ReactType;
```

Comment:
this does not match the current definition of 'componentShape`
Can you try and update the definition of component shape to
`export type componentShape = () => void | string | object;`
If that doesn't work we will investigate further.

Reply:
This should be updated to this 
```suggestion
  component?: React.ReactType<BreadcrumbHeadingProps>;
```
@tlabaj ComponetShape is no longer needed with typescript.  It refers to Proptypes.

Reply:
@tlabaj ... ahh component type might be needed but then we might have to create our own type.  let me check what this should be.

Reply:
I am confused are you saying  that React  ReactType  covers functions, object and string types?
(Note: ReactType is deprecated we should use ElementType)

Reply:
@jessiehuff did the above definition of componentShape not work?  What you have here will break JS consumers.